packages/beacon-node/test/unit/chain/validation/attestation/getShufflingForAttestationVerification.test.ts

```diff
@@ -0,0 +1,150 @@
+import {afterEach, beforeEach, describe, expect, it, vi} from "vitest";
```

Comment:
Spliced this file from `attestation.test.ts` as the nature of tests requires different mocks.